### DISCUSSION TITLE
Default Thumbnail Size for All Groups

### DISCUSSION THREAD

**Discussion by nextstep99** [fastaddons/GroupSpeedDial#294] [Q&A] [answered]
Is there a way to set a default thumbnail size -- say 480px x 360px -- so that they are the same across all groups? 

If yes, where can I find the option? (I've tried Options >>> Dials, and Options >>> Thumbnails - one of which is where I thought the setting might be)

Thanks!

**Comment by Juraj-Masiar** [comment, marked as answer]
Sadly, no.
When you create new group, it will copy properties of the current group, but if the groups are already created then they have independent values.
But this has been requested a few times already, I think I'll try to implement it somehow.
Just to clarify, you are using the "Dynamic" layout and you would like to apply same size for each of your groups, right?
I'll see what I can do!

**Reply by nextstep99** [reply]
Thank you for your clarification.

Yes. I'm using the dynamic layout. I'm using it on a widescreen monitor, divided into 3 vertical columns, (via windows  poertoys) which I sometimes use full screen.

From my point of view, I'd like to see a global default size setting under 'Dials" where the pixel height & px width and spacing can be set as default for all groups.

But each individual group could also set a custom size should someone wanted different sized dials as you have in Edit Groups. 

In that case Edit Groups there could also be an option for layout to have "Apply to All Groups" 

And also in Groups, maybe columns have a "Fluid" option, where the number of dials on display depends on their width, and the width of the monitor where the dials remain the same size but flow... so if when a widescreen monitor is divided into three screens, there might have just 5 dials in a single section/column, whereas as full width, the monitor might display 16 dials - without changing size. (Rather than the columns remain same and dials just increase in size)

I hope that make sense. Let me know if that needs clearer explanation, or if it's already a feature I just haven't found yet :)

I am just in the process of moving away from FVD, so I'm still experimenting on what can be done. 

Thank you for your excellent alternative dial and your quick response!
